app/main/views/briefs.py

```diff
@@ -154,10 +154,13 @@ def list_briefs():
             Lot.slug.in_(lot_slug.strip() for lot_slug in request.args["lot"].split(","))
         ))
 
+    response_headers = {"X-Compression-Safe": "0"}
     if request.args.get('status'):
-        briefs = briefs.has_statuses(
-            *(status.strip() for status in request.args['status'].split(','))
-        )
+        statuses = frozenset(status.strip() for status in request.args['status'].split(','))
+        briefs = briefs.has_statuses(*statuses)
+
+        if frozenset(("draft", "withdrawn",)).isdisjoint(statuses) and not with_users:
+            response_headers["X-Compression-Safe"] = "1"
```

Comment:
Will we need to also turn it off if the `with_users=True` param has been supplied?

Reply:
> `and not with_users:`

Indeed.

Reply:
🤦‍♀ Clearly not drunk enough coffee yet

Reply:
😂 😁